### DISCUSSION TITLE
tremor v1.1.0: New Donut Chart Component 🥳

### DISCUSSION THREAD

**Discussion by mitrotasios** [tremorlabs/tremor-npm#165] [Announcements]
We just released v1.1.0 that comes with a Donut Chart component as requested by many users! 🎉 
![release_github](https://user-images.githubusercontent.com/58337217/200063067-2b52a30e-91c2-49e9-98e7-66a77af563d3.png)

**Comment by gokulkrishh** [comment]
@1nfinite9 Quick question, i am not seeing legends for the donut chart variant, I am using 1.4.0 version. How to enable it?

**Reply by gokulkrishh** [reply]
I thought, there would be a default legend in donut chart, got it now. i have use `Legend` component instead.

**Reply by mitrotasios** [reply]
yup, sorry if this a bit unintuitive. We may add it to the actual component as well.

**Reply by gokulkrishh** [reply]
@mitrotasios Yep that would be great. otherwise loving the library. Keep rocking.

**Comment by qpre** [comment]
Nice ! is there a way to make it behave like a gauge ? (allowing to show a progress of X%)

**Reply by mitrotasios** [reply]
We don't have a gauge component per se, but you could use the donut chart to indicate progress I suppose 🙂
